testsuite/single/tests/crasher/1063-aux.c

```diff
@@ -0,0 +1,2 @@
+void create() {
```

Comment:
This test does nothing by it's own, but I need a different object I can load/destroy from the `1063.c` file.